README.md

```diff
@@ -13,6 +13,10 @@ While our first priority is a stable build, we will eventually add features that
 
 [Feature Requests](http://feathub.com/jellyfin/jellyfin)
 
+## Contributing to Jellyfin
+
+Please see our contributing documentation at https://github.com/jellyfin/jellyfin/blob/master/CONTRIBUTING.md
```

Comment:
Make it a nice link in Markdown? I also don't remember how to reference the latest version of that at all times...

Reply:
Oh I guess it is just blob/master. Okay, just a nicer link then.

"Please read CONTRIBUTING.md for details on how to submit pull requests to us."

Reply:
Same idea but different wording. Fixed.